app/views/TrusteesUkAddressView.scala.html

```diff
@@ -0,0 +1,72 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.routes._
+@import models.Mode
+
+@this(
+    main_template: MainTemplate,
+    formHelper: FormWithCSRF
+)
+
+@(form: Form[_], mode: Mode, index: Int, trusteeName: String)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = s"${errorPrefix(form)} ${messages("trusteesUkAddress.title", trusteeName)}"
+    ) {
+
+    @formHelper(action = TrusteesUkAddressController.onSubmit(mode, index), 'autoComplete -> "off") {
+
+        @components.back_link()
+
+        @components.error_summary(form.errors)
+
+        @components.heading("trusteesUkAddress.heading", optionalParam = Some(trusteeName))
```

Comment:
Does this need to be Some(trusteeName)? Can't we just pass trusteeName? I may not be understanding a change we have done here.

Reply:
optionalParam iof type Option[String] within component heading. If not provided then not passed into the message within heading component:

@(headingKey: String, optionalParam: Option[String] = None, headingSize: String = "heading-large")(implicit messages: Messages)
<h1 class="@headingSize">@messages(headingKey, if(optionalParam.isDefined) {optionalParam.get} else "")</h1>